decidim-comments/README.md

```diff
@@ -47,11 +47,10 @@ For the backend side:
 
 For the frontend side:
 
-- Typescript (introduced in #1001)
 - React
 - Apollo
```

Comment:
By the way @ferblape comments no longer work with React/Apollo, so these can be also removed.